app/models/course/condition.rb

```diff
@@ -14,6 +14,16 @@ class Course::Condition < ActiveRecord::Base
   ].freeze
 
   class << self
+    # Finds all the conditionals for the given course.
+    #
+    # @param [Course] course
+    # @return [Object] acts_as_conditionals objects belonging to the given course
```

Comment:
You might want to specify a common duck-typed method that all acts_as_conditional objects implement.